backend/src/v5/schemas/tickets/index.js

```diff
@@ -166,17 +167,30 @@ Tickets.validateTicket = async (teamspace, project, model, template, newTicket,
 			await generateModuleValidator(teamspace, project, model, template._id,
 				fullTem.modules, oldTicket?.modules, isNewTicket),
 		).default({}),
-		type: isNewTicket ? Yup.mixed().required() : Yup.mixed().strip(),
+		type: Yup.mixed().strip(),
 	};
 
+	if (isImport) {
+		if (template.config.comments) validatorObj.comments = Yup.array().min(1).of(importCommentSchema);
+		else if (newTicket.comments) {
+			throw new Error('Comments are not supported for this template type.');
+		}
+	}
+
 	const validatedTicket = await Yup.object(validatorObj).validate(newTicket, { stripUnknown: true });
 
 	// Run it again so we can check for unchanged properties that looked changed due to default values
 	validatorObj.modules = Yup.object(await generateModuleValidator(teamspace, project, model, template._id,
 		fullTem.modules, oldTicket?.modules, isNewTicket, true),
 	).default({});
 
-	return Yup.object(validatorObj).validate(validatedTicket, { stripUnknown: true });
+	const retVal = await Yup.object(validatorObj).validate(validatedTicket, { stripUnknown: true });
+
+	if (isNewTicket) {
```

Comment:
Depends what you mean by optimal. If you mean less lines of code? yes. But operation wise, we'd type check a value that either doesn't exist (import) or we've already type checked (single add/update) and we're always going to overwrite and transform them - i.e. we're doing a lot of meaningless operation around it.

Is it cleaner? also arguable, since i'd probably need to do something like `Yup.mixed().transform` and just assign a value.

I think the intention is clearer this way and less operations - I know the template we used to validate the data and i'm making sure it's in there as the type if it's a new ticket.